### PR TITLE
ET-988 precommit hook with gitleaks

### PR DESCRIPTION
**Pull request by Hanna10Domnich**

### CODE REVIEW

File: gitleaks.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+gitleaksEnabled=$(git config --bool hooks.gitleaks)
+cmd="gitleaks --repo-path=./ --verbose --redact --pretty"
+if [ $gitleaksEnabled = "true" ]; then
+    $cmd
+    if [ $? -eq 1 ]; then
+cat <<\EOF
+Error: gitleaks has detected sensitive information in your changes.
+If you know what you are doing you can disable this check using:
+    git config hooks.gitleaks false
```
Comment (andrei-oksentyk) [inline gitleaks.sh:12]: It's never mind and can be removed. Simpler to use `--no-verify` flag instead of changing git config.

File: gitleaks.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# This is an example of what adding gitleaks to a pre-commit hook would look like.
```
Comment (andrei-oksentyk) [inline gitleaks.sh:2]: Remove this line and/or add desription of what this script does

File: gitleaks.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#script that detects hardcoded secrets and prevents pushing them to repo
```
Comment (juliamaksimchik) [inline gitleaks.sh:1]: Please locate this file as suggested in instructions

Screen Shot 2020-06-01 at 10 41 10

Comment (Hanna10Domnich) [reply gitleaks.sh:1]: It's a great idea, but for now, I think It's better for us that we can disable it with flags.
cause now we have 96 leaks and we are blocked by this tool until we fix it.

**Pull request closed without merging**